Mathlib/Data/Nat/Cast/Basic.lean

```diff
@@ -23,8 +23,9 @@ assert_not_exists OrderedCommGroup
 assert_not_exists Commute.zero_right
 assert_not_exists Commute.add_right
 assert_not_exists abs_eq_max_neg
-assert_not_exists natCast_ne
-assert_not_exists MulOpposite.natCast
+assert_not_exists NeZero.natCast_ne
+-- `MulOpposite.op_natCast` was not intended to be imported
```

Comment:
So this `assert_not_exists` fails now, right? Can you make the comment mention "TODO" somewhere?

Reply:
TODO added, thanks!